celery/bin/worker.py

```diff
@@ -166,8 +166,8 @@ def detach(path, argv, logfile=None, pidfile=None, uid=None,
               type=LOG_LEVEL,
               help_group="Worker Options",
               help="Logging level.")
-@click.option('optimization',
-              '-O',
+@click.option('-O',
```

Comment:
-O before or after? just for double check

Reply:
Before :)

Reply:
See code above/below, the pattern is consistent being first `-O`, then `--performance`